app/templates/components/event-card.hbs

```diff
@@ -15,17 +15,21 @@
       </a>
     {{/unless}}
     <div class="main content">
-      <a class="header" href="{{href-to 'public' event.identifier}}">
-        <span>{{event.name}}</span>
-      </a>
+      {{#smart-overflow}}
+        {{#link-to 'public' event.identifier class='ui header'}}
```

Comment:
And any specific reason you have used the link-to component here? What advantage does this give us?